### PR TITLE
Standardized API's

### PR DESCRIPTION
**Pull request by bhavanakarwade**
## What ## 

- Refactored the API endpoints in the User, Organization, Auth and Agent modules.
- Improved code Organization and readability.
- Updated API documentation.

## Why ##

- To provide better support for future feature development.
- To make the API more user-friendly and intuitive for developers.

## How ##

- Reviewed the existing Codebase in the User, Organization, Auth and Agent modules.
- Refactored the API endpoints using best practices and design patterns.
- Updated API documentation to reflect the changes made.
- Ran tests to ensure that the refactored code functions correctly.

### CODE REVIEW

File: src/api/Auth.ts [hunk 1]
```diff
@@ -12,7 +12,8 @@ export interface UserSignUpData {
     email: string,
 }
 export interface AddPasswordDetails {
-    password?:string
+    email:string
+    password:string | undefined
```
Comment (nishad-ayanworks) [inline src/api/Auth.ts:16]: What is the use to set `undefined`

File: src/api/Fido.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const generateRegistrationOption = async (payload: RegistrationOptionInte
   }
 }
 
-export const verifyRegistration = async (payload:unknown, userEmail:string) => {
+export const verifyRegistration = async (payload:any, userEmail:string) => {
```
Comment (nishad-ayanworks) [inline src/api/Fido.ts:30]: Try not to use `any` data type.

File: src/components/Profile/EditUserProfile.tsx [hunk 3]
```diff
@@ -152,7 +152,7 @@ const UpdateUserProfile = ({ toggleEditProfile, userProfileInfo, updateProfile }
     setLoading(true)
 
     const userData = {
-      id: userProfileInfo?.id,
+      // id: userProfileInfo?.id,
```
Comment (nishad-ayanworks) [inline src/components/Profile/EditUserProfile.tsx:155]: please remove commented code

**Review by nishad-ayanworks — CHANGES_REQUESTED** [review]
Please resolve the comments

File: src/components/Profile/EditUserProfile.tsx (hunk 3 above)
Comment (bhavanakarwade) [reply src/components/Profile/EditUserProfile.tsx]: Resolved

File: src/api/Auth.ts (hunk 1 above)
Comment (bhavanakarwade) [reply src/api/Auth.ts]: Resolved

**Comment by bhavanakarwade** [comment]
> Please resolve the comments

Resolved

**Review by nishad-ayanworks — APPROVED** [review]
LG

**Pull request merged**